tools/clang-tidy-plugin/CMakeLists.txt

```diff
@@ -55,6 +58,7 @@ if (CATA_CLANG_TIDY_EXECUTABLE)
 else ()
     set(CataAnalyzerName CataAnalyzerPlugin)
     add_library(${CataAnalyzerName} MODULE ${CataAnalyzerSrc})
+    target_link_libraries(${CataAnalyzerName} PRIVATE clangTidyPlugin)
```

Comment:
This does nothing, sadly. As in, the dll builds, but it does not get picked up by clang-tidy. I have not investigated why yet. But it would be so cool if it works!!!